pkg/events/events.go

```diff
@@ -37,20 +48,20 @@ func WarningEventf(recorder record.EventRecorder, object runtime.Object, reason,
 
 // RemediationStarted will record a Normal event with reason RemediationStarted and message "Remediation started".
 func RemediationStarted(recorder record.EventRecorder, object runtime.Object) {
-	NormalEvent(recorder, object, "RemediationStarted", "Remediation started")
+	NormalEvent(recorder, object, RemediationStartedEventReason, RemediationStartedEventMessage)
 }
 
 // RemediationStoppedByNHC will record a Normal event with reason RemediationStopped and message "NHC added the timed-out annotation, remediation will be stopped".
 func RemediationStoppedByNHC(recorder record.EventRecorder, object runtime.Object) {
-	NormalEvent(recorder, object, "RemediationStopped", "NHC added the timed-out annotation, remediation will be stopped")
+	NormalEvent(recorder, object, RemediationStoppedEventReason, RemediationStoppedEventMessage)
 }
 
 // RemediationFinished will record a Normal event with reason RemediationFinished and message "Remediation finished".
 func RemediationFinished(recorder record.EventRecorder, object runtime.Object) {
-	NormalEvent(recorder, object, "RemediationFinished", "Remediation finished")
+	NormalEvent(recorder, object, RemediationFinishedEventReason, RemediationFinishedEventMessage)
 }
 
 // GetTargetNodeFailed will record a Warning event with reason RemediationFailed and message "Could not get remediation target node".
 func GetTargetNodeFailed(recorder record.EventRecorder, object runtime.Object) {
-	WarningEvent(recorder, object, "RemediationCannotStart", "Could not get remediation target Node")
+	WarningEvent(recorder, object, RemediationCannotStartEventReason, RemediationCannotStartEventMessage)
```

Comment:
this is very inconsistent :/ 

func name: GetTargetNodeFailed
reason: RemediationCannotStart
message: var name RemediationCannotStart, but value about the target node

IMHO most useful would be:
func name: RemediationCannotStart
reason: RemediationCannotStart
msg: passed in by parameter, there probably can be multiple reasons for a remediation to not start

*Changing* this would be an API change though, which would, when done correctly, need a major version bump...

So my suggestion is:
- rename the msg var to e.g. RemediationCannotStartTargetNodeFailedEventMessage , to make its meaning a bit clearer
- *add* a function as suggested above, and use it inside GetTargetNodeFailed?

WDYT?

Also, while I agree that the reasons should be consistent and might be worth adding to the API, I don't see that for the messages. What's the motivation for making them public?

Reply:
> What's the motivation for making them public?

Mostly testing, but I'm still torn about leaving this information available only for testing and I won't mind reverting this particular bit :shrug: 

> this is very inconsistent :/

I agree. I dunno why I choose a function name so different than the others :/

> rename the msg var to e.g. RemediationCannotStartTargetNodeFailedEventMessage , to make its meaning a bit clearer
etc.

sounds like a good idea :)